scripts/install/authelia.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+#
+if [[ ! -f /install/.nginx.lock ]]; then
+    echo_warn "You must install nginx first"
+    exit
+fi
+
+mkdir -p "/opt/authelia"
+wget -qO "/opt/authelia.tar.gz" "https://github.com$(/usr/bin/curl -sNL https://github.com/authelia/authelia/releases | grep -Eom1 '/authelia/(.*)tar.gz')"
```

Comment:
Would this be something we might want to pin the version of btw? Like specifically the last one we tested for example